sample-customisation/src/main/java/net/helpscout/sample/beacon/CustomisationApplication.java

```diff
@@ -1,35 +1,36 @@
 package net.helpscout.sample.beacon;
 
 import android.app.Application;
-
 import com.helpscout.beacon.Beacon;
 import com.helpscout.beacon.ui.BeaconEventLifecycleHandler;
 import com.helpscout.beacon.ui.BeaconOnClosedListener;
 import com.helpscout.beacon.ui.BeaconOnOpenedListener;
-
 import timber.log.Timber;
 
 public class CustomisationApplication extends Application {
 
-    private static String secureBeaconId = "YOUR_BEACON_ID";
+    // TODO replace with your Beacon Id from Help Scout
+    private static String beaconId = "6563caf4-4fbe-4e27-924e-20e088d2ce81";
```

Comment:
❓ Is this a test `beaconId` that we've created specifically for this? Should we provide a default id in the first place? When the app starts up, we could check if a `beaconId` is set and throw an exception providing a meaningful message instead if not, so that they know that they need to set a `beaconId` for the sample project to work.

Reply:
Ah that's an accidental check-in 🤦 , I should have changed it to `""` as it in with the Simple Kotlin project so it'll runtime crash if the developer hasn't added their Beacon ID - Logcat has a very obivous message to highlight the lack of Beacon Id as the reason for the crash.